src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -1529,6 +1529,30 @@ public static IEdmTypeDefinitionReference GetUInt64(this EdmModel model, string
                 isNullable);
         }
 
+        /// <summary>
+        /// Get the <see cref="IEdmTargetPath"/> from a target path string.
+        /// </summary>
+        /// <param name="model">The EdmModel.</param>
+        /// <param name="targetPath">Target path containing segments separated by '/'. For example: "A.B/C/D.E/Func1(NS.T,NS.T2)/P1".</param>
+        /// <param name="caseInsensitive">Property name case-insensitive or not.</param>
+        /// <returns>The created <see cref="IEdmTargetPath"/>.</returns>
+        public static IEdmTargetPath GetTargetPath(this IEdmModel model, string targetPath, bool caseInsensitive = true)

```

Comment:
>GetTargetPath [](http://example.com/codeflow?start=37&length=13)

Add at lease extension method like ( or similar) :

public static string GetFullName(this IEdmTargetPath targetPath)

Reply:
Added an extension method `GetPathString`